### DISCUSSION TITLE
Getting Trojan error during download of 10.0

### DISCUSSION THREAD

**Discussion by VX5Cueball** [Khopa/dcs_liberation#3328] [Dev Diaries]
Are we aware of this or not? Every time I try to download and halfway through a huge alarm goes of in my protection software. I want to run this but hitting this wall.

**Comment by Starfire13** [comment]
False positives like this are not uncommon with apps compiled by Github, particularly with certain antivirus software.

**Reply by VX5Cueball** [reply]
Was just checking before proceeding. I have an older  version of liberation and wanted to get the newer one. Thank you.
